crates/test-utils/src/rpc.rs

```diff
@@ -34,7 +34,10 @@ static RETH_HOSTS: LazyLock<Vec<&'static str>> = LazyLock::new(|| {
 
 // List of general purpose DRPC keys to rotate through
 static DRPC_KEYS: LazyLock<Vec<String>> = LazyLock::new(|| {
-    let mut keys = vec!["AgasqIYODEW_j_J0F91L8oETmhtHCXkR8JAVssvAG40d".to_owned()];
+    let mut keys = vec![
```

Comment:
I see that the tests are still failing because of invalid API keys. Let's try to debug this a bit.

Comment the `if` block which starts on line 42